packages/components/carousel/src/carousel.vue

```diff
@@ -80,6 +80,7 @@ import { ElIcon } from '@element-plus/components/icon'
 import { ArrowLeft, ArrowRight } from '@element-plus/icons-vue'
 import { useNamespace } from '@element-plus/hooks'
 import { carouselContextKey } from '@element-plus/tokens'
+import { useOrderedChildren } from '@element-plus/hooks/use-ordered-children'
```

Comment:
We could export composable function at the entry of hooks.

`@element-plus/hooks/index.ts`
```ts
export * from './use-ordered-children'
```